ci/checkstyle.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0"?>
```

Comment:
We have a section in our wiki with 2 suggested styles that we use, can we use the "Android Style with Improvements" style ? https://github.com/LateralView/android-base-project/wiki/Android-Studio-Configuration ?

Reply:
That style file applies to the IDE, and it has different format.
Try to reformat the code in this branch (Right click on main folder -> Reformat code) and let me know if you see any differences.

I took the checkstyle that Oracle is using and applied some changes

Reply:
Oh, I see, we need a specific xml file to set our checkstyle and the one that we use with the IDE is not compatible with this, right ? I am very sure that the checkstyle that you take from Oracle doesn't have the changes that I made on the Android recommended styles, like the life cycle arrangements, etc.

Reply:
They are not compatible.
The easiest way is to `Reformat` this code and see what changes.